src/ngraph/frontend/onnx_import/utils/reshape.cpp

```diff
@@ -59,9 +60,8 @@ namespace ngraph
 
             AxisVector get_default_axis_vector(std::size_t data_shape_size, std::size_t start_value)
             {
-                AxisVector axis_vector(data_shape_size);
-                std::iota(std::begin(axis_vector), std::end(axis_vector), start_value);
-                return axis_vector;
+                return AxisVector{onnx_import::common::get_monotonic_range<std::size_t>(
```

Comment:
`common::get_monotonic_range<>` unless this piece of code is not part of `onnx_import` namespace.

Reply:
and `return {...}`, the `AxisVector` is not required.

Reply:
Does adding `AxisVector` hurt anything? If not then, it makes the code more explicit and less magical...

Reply:
I think that returning `AxisVector{...}` is more readable.